src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -1078,6 +1080,60 @@ class spell_warr_heroic_leap_jump : public SpellScriptLoader
     }
 };
 
+// Shockwave - 46968
+class spell_warr_shockwave : public SpellScriptLoader
+{
+public:
```

Comment:
it's not a tab problem, just use the same indentation like the other scripts

btw alphabetical order will be great